api/app/Exceptions/ValidationException.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace App\Exceptions;
+
+use Illuminate\Support\MessageBag;
+use Illuminate\Http\Exception\HttpResponseException;
+
+class ValidationException extends HttpResponseException
```

Comment:
@artstorm ah I think you misunderstood - see in the `use` commands in the example above - there is two different namespaces - one from `Illuminate` the other from `Symfony`. It is the `Illuminate\Http\Exception\HttpResponseException` that has the `getResponse` method.